rfcs/document_all_media_properties/20230307-project_proposal.md

```diff
@@ -0,0 +1,133 @@
+Project proposal for
+[Document all media properties](https://github.com/WordPress/openverse/issues/412)
+
+## Due date:
+
+2023-03-21
+
+## Assigned reviewers
+
+- [ ] @AetherUnbound
+- [ ] @krysal
+
+## Description
+
+### Year Goal
+
+Overcoming **Data Inertia**.
+
+### Motivation
+
+The first step in making the data we have reliable is describing it in detail.
+Currently, we have more than 600 million rows of data about media in the catalog
+database and similar but slightly different sets of data in the API database and
+elasticsearch. Because of the history of the project, some data we have does not
+conform to the requirements we now set and does not have the fields we describe
+as required, and there are some fields that store a very different data piece
+than what the users might expect[^1].
+
+The main goal of this project is to establish a comprehensive baseline of our
+media data: what information we currently have about each media item and what we
+expect to have. Our database currently holds more than 700 million rows of media
+data. The same data is in our API database and Elasticsearch, but the data
+properties there are slightly different. Due to the project's history, some data
+in the catalog does not meet our current requirements and lacks necessary
+fields, while other fields store data that doesn't align with user
+expectations[^1].
```

Comment:
This looks to be a rephrasing of the prior paragraph?

Reply:
Oops, you're right! I'll remove the first paragraph.